src/test/java/com/amazon/dlic/auth/http/jwt/HTTPJwtAuthenticatorTest.java

```diff
@@ -534,6 +535,306 @@ public void testRequiredIssuerWithIncorrectAudience() {
+        keyGen.initialize(2048);
+        KeyPair pair1 = keyGen.generateKeyPair();
+        PrivateKey priv1 = pair1.getPrivate();
+        PublicKey pub1 = pair1.getPublic();
+
+        KeyPair pair2 = keyGen.generateKeyPair();
+        PrivateKey priv2 = pair2.getPrivate();
+        PublicKey pub2 = pair2.getPublic();
+
+        String jwsToken1 = Jwts.builder().setSubject("Leonard McCoy").signWith(priv1, SignatureAlgorithm.RS256).compact();
+        String jwsToken2 = Jwts.builder().setSubject("Stephen Crawford").signWith(priv2, SignatureAlgorithm.RS256).compact();
+
+        Settings settings = Settings.builder()
+            .put(
+                "signing_key",
+                "-----BEGIN PUBLIC KEY-----\n"
+                    + BaseEncoding.base64().encode(pub1.getEncoded())
+                    + "-----END PUBLIC KEY-----,-----BEGIN PUBLIC KEY-----\n"
+                    + BaseEncoding.base64().encode(pub2.getEncoded())
+                    + "-----END PUBLIC KEY-----"
+            )
+            .build();
+
+        HTTPJwtAuthenticator jwtAuth = new HTTPJwtAuthenticator(settings, null);
+        Map<String, String> headers1 = new HashMap<String, String>();
+        headers1.put("Authorization", "Bearer " + jwsToken1);
+
+        AuthCredentials creds1 = jwtAuth.extractCredentials(
+            new FakeRestRequest(headers1, new HashMap<String, String>()).asSecurityRequest(),
+            null
+        );
+
+        Assert.assertNotNull(creds1);
+        assertThat(creds1.getUsername(), is("Leonard McCoy"));
+        assertThat(creds1.getBackendRoles().size(), is(0));
+
+        Map<String, String> headers2 = new HashMap<String, String>();
+        headers2.put("Authorization", "Bearer " + jwsToken2);
+        AuthCredentials creds2 = jwtAuth.extractCredentials(
+            new FakeRestRequest(headers2, new HashMap<String, String>()).asSecurityRequest(),
+            null
+        );
+
+        Assert.assertNotNull(creds2);
+        assertThat(creds2.getUsername(), is("Stephen Crawford"));
+        assertThat(creds2.getBackendRoles().size(), is(0));
+    }
+
+    @Test
+    public void testMultipleSigningKeysParseWithSpacesSuccessfully() throws NoSuchAlgorithmException {
+
+        KeyPairGenerator keyGen = KeyPairGenerator.getInstance("RSA");
+        keyGen.initialize(2048);
+        KeyPair pair1 = keyGen.generateKeyPair();
+        PrivateKey priv1 = pair1.getPrivate();
+        PublicKey pub1 = pair1.getPublic();
+
+        KeyPair pair2 = keyGen.generateKeyPair();
+        PrivateKey priv2 = pair2.getPrivate();
+        PublicKey pub2 = pair2.getPublic();
+
+        String jwsToken1 = Jwts.builder().setSubject("Leonard McCoy").signWith(priv1, SignatureAlgorithm.RS256).compact();
+        String jwsToken2 = Jwts.builder().setSubject("Stephen Crawford").signWith(priv2, SignatureAlgorithm.RS256).compact();
+
+        Settings settings = Settings.builder()
+            .put(
```

Comment:
Does this set need to be updated to use `putList` instead of passing a command separated string?

Reply:
The tests pass so I think it works either way. I just copied the pattern from the required_audience field which already used a list.